scripts/create-jwt.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Function to convert a string to base64 URL encoding
+function b64url() { 
+    echo "${1}" | openssl base64 -a -A | tr '+' '-' | tr '/' '_' | tr -d '='
```

Comment:
```suggestion
    openssl base64 -A <<<"${1}" | tr +/ -_ | tr -d =
```
It's much easier if `gbasenc` is available, but that requires `brew install coreutils`.